src/gui/guiFormSpecMenu.cpp

```diff
@@ -3423,7 +3380,7 @@ void GUIFormSpecMenu::updateSelectedItem()
 
 	// If craftresult is nonempty and nothing else is selected, select it now.
 	if (!m_selected_item) {
-		for (const GUIInventoryList *e : m_inventorylists) {
+		for (const irr_ptr<GUIInventoryList> &e : m_inventorylists) {
```

Comment:
Not that there's a change here.
`const irr_ptr<T> e` is like `T *const e`, `irr_ptr<const T> e` would be like `const T *e`.
I could add some `const`s everywhere, if requested. (Ew, mutability by default.)